libgalois/src/analytics/subgraph_extraction/subgraph_extraction.cpp

```diff
@@ -67,6 +108,10 @@ SubGraphNodeSet(
       katana::steal(), katana::no_stats(),
       katana::loopname("SubgraphExtraction"));
 
+  for (uint64_t i = 0; i < num_nodes; ++i) {
+    katana::gPrint("out", (*out_indices)[i]);
+  }
+
```

Comment:
Don't print in the algorithm. Also try to use `KATANA_LOG_DEBUG` and friends. That's the modern interface.